src/sentry/static/sentry/app/views/projects/redirectSentry9Project.jsx

```diff
@@ -0,0 +1,157 @@
+import React from 'react';
+import {withRouter} from 'react-router';
+import PropTypes from 'prop-types';
+import _ from 'lodash';
+
+import {t} from 'app/locale';
+import withApi from 'app/utils/withApi';
+import LoadingError from 'app/components/loadingError';
+
+// TODO: This is react-router v4 <Redirect to="path/" /> component to allow things
+//       to be declarative
+export class Redirect extends React.Component {
+  static propTypes = {
+    router: PropTypes.object.isRequired,
+    to: PropTypes.oneOfType([PropTypes.string, PropTypes.object]).isRequired,
+  };
+
+  componentDidMount() {
+    this.props.router.replace(this.props.to);
+  }
+
+  render() {
+    return null;
+  }
+}
+
+class ProjectDetailsInner extends React.Component {
+  static propTypes = {
+    api: PropTypes.object.isRequired,
+
+    orgId: PropTypes.string.isRequired,
+    projectId: PropTypes.string.isRequired,
+  };
+
+  state = {
+    loading: true,
+    error: null,
+    project: null,
+  };
+
+  componentDidMount() {
+    this.fetchData();
```

Comment:
We should have some analytics to see how often people still access this.

Reply:
@billyvg is there some analytics code I should be adding here?

Reply:
Check out `app/utils/analytics`